plugins/module_utils/netbox_utils.py

```diff
@@ -1076,6 +1076,11 @@ def _build_query_params(
                 query_dict.update(
                     {"interface_id": module_data.get("assigned_object_id")}
                 )
+            elif module_data["assigned_object_type"] == "ipam.fhrpgroup":
+                query_dict.update(
+                    {"???": module_data.get("assigned_object_id")}
```

Comment:
I'm just guessing here but it _might_ be `group_id`. It's hard to tell, looking at https://github.com/netbox-community/netbox/blob/c660f1c019ddb46713f830c360dd5804784c73df/netbox/ipam/api/serializers_/fhrpgroups.py#L5.